pip/update.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+import os
+
+update_list = []
+p = os.popen('sudo -H pip freeze --local',"r")
+while 1:
+    line = p.readline()
+    if not line: break
```

Comment:
I'll do some work on making this universal for Windows also, I'm sure I could do OS checks and just split the nix and nt code as functions.

I just added this in the fork to get some feedback as I wrote this a couple years back and decided hey why not actually introduce it to the community.

I'll do some work on it and report back here and see what everyone thinks